services/ui-src/src/components/fields/TextAreaField.test.tsx

```diff
@@ -7,7 +7,7 @@ jest.mock("react-hook-form", () => ({
   useFormContext: () => ({
     setValue: () => {},
     register: () => {},
-    getValues: jest.fn().mockReturnValue({}),
+    getValues: jest.fn().mockReturnValueOnce([]).mockReturnValue("test"),
```

Comment:
Fixes `Warning: Failed prop type: Invalid prop `value` supplied to `TextInput`, expected one of type [string, number].`.